Osiris/Hacks/Backtrack.cpp

```diff
@@ -108,8 +112,53 @@ void Backtrack::run(UserCmd* cmd) noexcept
     if (bestRecord) {
         auto record = records[bestTargetIndex][bestRecord];
         memory->setAbsOrigin(bestTarget, record.origin);
-        cmd->tickCount = timeToTicks(record.simulationTime + getLerp());
+        cmd->tickCount = timeToTicks(record.simulationTime);
+    }
+}
+
+void Backtrack::AddLatencyToNetwork(NetworkChannel* network, float latency) noexcept
+{
+    for (auto& sequence : sequences)
+    {
+        if (memory->globalVars->serverTime() - sequence.servertime >= latency)
+        {
+            network->InReliableState = sequence.inreliablestate;
+            network->InSequenceNr = sequence.sequencenr;
+            break;
+        }
+    }
+}
+
+void Backtrack::UpdateIncomingSequences(bool reset) noexcept
+{
+    static float lastIncomingSequenceNumber = 0.f;
+
+    if (reset)
```

Comment:
Remove reset, it doesnt work good

Reply:
removing the whole 
```
    if (reset)
        lastIncomingSequenceNumber = 0.f;
```
?

can you elaborate why, i am not an expert on this code, but id love to learn, this is interesting. :)

Reply:
remove reset, since
we check
`if(network->InSequenceNr > lastIncomingSequenceNumber)`
if lastIncomingSequenceNumber becomes way bigger than InSequenceNr it wont update ping (i used reset to set lastIncomingSequenceNumber to 0, but checking if it isnt equal works better)

Reply:
Nice, ok i see now. :)

Reply:
@notgoodusename madbruv#7387